### DISCUSSION TITLE
Welcome to AU Core Discussions!

### DISCUSSION THREAD

**Discussion by brettesler-ext** [hl7au/au-fhir-core#4] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Upvote the FHIR Resources that should be profiled in Release 1 of AU Core for your products so we can work on what we need to
  * Ask questions you’re wondering about.
  * Share ideas on future work and other feature requests so we can discuss as a community.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

#### When to use GitHub Discussions

GitHub Discussions is where we will conduct polls and other voting, and is best suited as a tool to share questions, ideas, conversations, requests for comment (RFC), resource planning, and announcements.

Ideas on future work or feature requests should start as discussions.

#### When to use GitHub Issues

GitHub Issues are used to track executable pieces of work with a defined start and end point. These can include new features, fixing bugs, general updates, and tracking for epics and sprints, among other things. Often an issue started life as a discussion and at a certain point we have enough consensus to create an issue and proceed.

[We can create an issue directly from a discussion](https://docs.github.com/en/issues/tracking-your-work-with-issues/creating-an-issue#creating-an-issue-from-discussion), which means you can attach your notes and decisions that led to a piece of a work directly in an Issue.

We can also convert issues into discussions if something needs further discussion.

**Comment by mjosborne1** [comment]
Hi All, 
   I am Michael Osborne and I have recently joined CSIRO and the Sparked FHIR Accelerator project as a FHIR Terminology specialist.

**Comment by dt-r** [comment]
Hi All 👋, I am Danielle Tavares-Rixon. I am a co-chair for the AU Core TDG and a member of CSIRO as part of the Sparked FHIR Accelerator.

**Comment by dylan-jf7** [comment]
Hi All,
I am Dylan Fernandes a solution architect working for Telstra Health in the hospitals space

**Comment by dbojicic** [comment]
Hi All, 
I am Dusica Bojicic. I've recently joined CSIRO and the Sparked FHIR Accelerator project as a FHIR Standards Developer.
